reference/6/PowerShellGet/Get-InstalledModule.md

```diff
@@ -66,8 +75,10 @@ Accept wildcard characters: False
 ```
 
 ### -AllVersions
+
 Indicates that you want to get all available versions of a module.
-You cannot use the *AllVersions* parameter with the *MinimumVersion*, *MaximumVersion*, or *RequiredVersion* parameters.
+You cannot use the **AllVersions** parameter with the *MinimumVersion*, *MaximumVersion*, or
```

Comment:
MinimumVersion and MaximumVersion missing bold.

Reply:
fixed